packages/@glimmer/manager/test/package.json

```diff
@@ -1,9 +1,12 @@
 {
   "private": true,
   "name": "@glimmer-test/manager",
+  "repo-meta": {
+    "strictness": "loose"
+  },
   "dependencies": {
-    "@glimmer/manager": "workspace:*",
     "@glimmer/interfaces": "workspace:*",
+    "@glimmer/manager": "workspace:*",
```

Comment:
There's a decent amount of missing dependencies in test packages, which worked kind of by accident.
It's nice to document things properly.